### PR TITLE
Embed timestamp

### PR DESCRIPTION
**Pull request by PapauloGamerOfc**
Please do NOT use `Merge Squash`.

### CODE REVIEW

**Comment by mlite** [comment]
It looks good to me.

**Review by mlite — APPROVED** [review]

**Comment by mlite** [comment]
I will let @LucasMull to check and merge it.

**Review by LucasMull — APPROVED** [review]
Looks good to me!

File: orka-utils.c [hunk 1]
```diff
@@ -142,8 +142,9 @@ orka_unix_ms_to_iso8601(char *str, size_t len, void *p_data)
 
   struct tm *tm = localtime(&seconds);
 
+  // @todo find a better solution, like fix json_inject and use (timestamp):|F|
   return snprintf(str, len,
-    "%d-%.2d-%dT%.2d:%.2d:%.2d.%.3dZ", // ISO-8601 complete format
+    "\"%d-%.2d-%dT%.2d:%.2d:%.2d.%.3dZ\"", // ISO-8601 complete format
     tm->tm_year + 1900, tm->tm_mon + 1, tm->tm_mday, // Date
```
Comment (mlite) [inline orka-utils.c:148]: This change is no longer needed. I will add the support of `|F|`

**Pull request merged**